File: Matlab/fung.m
% Calculating the dem=nominator in Racah Formula

function r = fung(t, j1,j2,j3,J1,J2,J3)

r = factorial(t-j1-j2-j3).*factorial(t-j1-J2-J3).*factorial(t-J1-j2-J3).*factorial(t-J1-J2-j3).*factorial(j1+j2+J1+J2-t).*...
   factorial(j2+j3+J2+J3-t).*factorial(j3+j1+J3+J1-t);

File: Julia/fung.jl
#Calculating the denominator in Racah Formula

function fung(tt,j1,j2,j3,J1,J2,J3)

  return r = factorial(tt-j1-j2-j3)*factorial(tt-j1-J2-J3)*factorial(tt-J1-j2-J3)*factorial(tt-J1-J2-j3)*factorial(j1+j2+J1+J2-tt)*
   factorial(j2+j3+J2+J3-tt)*factorial(j3+j1+J3+J1-tt)
 end
